### PR TITLE
ensure to add trailing slash at the end of relative paths and the ff …

### PR DESCRIPTION
**Pull request by sumitshinde-84**
…signup page

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by joepavitt — APPROVED** [review]

File: .eleventy.js [hunk 1]
```diff
@@ -670,25 +670,30 @@ module.exports = function(eleventyConfig) {
...
+            }
+    
+            // Ensure the URL has a trailing slash, but do not update if it contains a '#' or ends with '.md' or https
+            if (!href.endsWith('/') && !href.includes('#') && !href.endsWith('.md') && !href.includes('https')) {
+                href += '/';
+            }
+    
+            // make sure to update the flowfuse signup URL
+            if (href.includes('https://app.flowfuse.com/')) {
+                href += '/';
```
Comment (knolleary) [inline .eleventy.js:690]: @sumitshinde-84 isn't this going to be adding an extra `/` regardless of whether it already ends with `/`? You've added this outside of any check to guard against that.

Comment (joepavitt) [reply .eleventy.js:690]: good point - I miss that

Comment (sumitshinde-84) [reply .eleventy.js:690]: Yes, I should first check if it already has the trailing slash or not, but I also want to make sure that the host, like app.flowfuse.com, will not have the slash added at the end.

Comment (sumitshinde-84) [reply .eleventy.js:690]: i will fix it now !

**Pull request merged**